Frontend/src/components/Navbar.tsx

```
@@ -0,0 +1,186 @@
+            <div className='hover:cursor-pointer'>
+                <img className='w-20 rounded-full' src="/images/courier logo.png" alt="logo icon" />
+            </div>
+            <div>
+                <h1 className='font-medium '>Self Courier</h1>
+            </div>
+            
+        </div>
+        {/* nav items */}
+        <ul className='ml-5 flex space-x-2 items-center'>
+            {/* Home */}
+            <li><a href="">Home</a></li>
```

Biome (1.9.4)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)